Sample/LoginViewController.swift

```diff
@@ -143,7 +143,7 @@ extension LoginViewController {
 }
 
 final class MyChatChannelListRouter: ChatChannelListRouter {
-    override func showMessageList(for cid: ChannelId) {
+    override func showChannel(for cid: ChannelId) {
```

Comment:
regarding naming: it should be consistent across all the places, 
its just a note to double-check
if its a message list, it should be always: MessageListModel, MessageListViewController, MessageListView, etc

@petyosi could you please check how this name across platforms, so we could be consistent

Reply:
The `MessageList` is the component to render a list of messages, just like the name describes it. But the message list can be rendered in a Channel or in a Thread. In this case, we are showing a channel (with a message list).

Reply:
Do we have a difference between Channel/Tread/MessageList somewhere documented?

Reply:
Not sure, but it seems pretty obvious. You open a channel, you have a list of messages, you open a thread, you have a list of messages. This is the aim of this PR, is to reuse the `messageList` component between the Channel and Thread, before it was duplicated code